lighthouse-core/report/report-generator.js

```diff
@@ -202,10 +202,21 @@ class ReportGenerator {
 
   /**
    * Gets the script for the report UI
-   * @return {string}
+   * @param {string} reportContext
+   * @return {Array<string>}
    */
-  getReportJS() {
-    return fs.readFileSync(path.join(__dirname, './scripts/lighthouse-report.js'), 'utf8');
+  getReportJS(reportContext) {
+    const scriptList = [];
+
+    if (reportContext === 'perf-x') {
+      scriptList.push(fs.readFileSync(path.join(__dirname, './scripts/perf-x-api.js')));
+    }
+
+    if (reportContext !== 'devtools') {
+      scriptList.push(fs.readFileSync(path.join(__dirname, './scripts/lighthouse-report.js')));
```

Comment:
Sorry. I didn't add this when I rewrite this method because I thought 'utf8' is default anyway : (
I will add it back in the next commit. Would you mind to tell me why it's necessary though?